spacemacs/config.el

```diff
@@ -168,6 +168,12 @@ Can be installed with `brew install trash'."
 ;; Single space between sentencs is more widespread than double
 (setq-default sentence-end-double-space nil)
 
+;; The C-d rebinding that most shell-like buffers inherit from
+;; comint-mode assumes non-evil configuration with its
+;; `comint-delchar-or-maybe-eof' function, so we disable it
+(with-eval-after-load 'comint
```

Comment:
I see. Then, why didn't Travis catch it? Should we change the `.travis.yml` to test against Emacs 24.3?